admin/app/controllers/spree/admin/invitations_controller.rb

```
@@ -0,0 +1,108 @@
+          render :new, status: :unprocessable_entity
+        end
+      end
+
+      # GET /admin/invitations/:id?token=:token
+      def show
+        @invitation = Spree::Invitation.pending.not_expired.find_by!(id: params[:id], token: params[:token])
+
+        # if there's no current user, redirect to the new admin user path so they can sign up
+        unless try_spree_current_user
+          redirect_to spree.new_admin_admin_user_path(token: @invitation.token), status: :see_other
+          return
```

RuboCop (1.73)

[convention] Redundant `return` detected.

(Style/RedundantReturn)